analyses/molecular-subtyping-chordoma/01-Subtype-chordoma.Rmd

```diff
@@ -141,16 +133,15 @@ Look at SMARCB1 expression values only in chordoma
 ```{r}
 # get the row that contains the SMARCB1 values
 # gene symbols are rownames
-smarcb1_expression <- expression_data[which(rownames(expression_data) == "SMARCB1"), ]
-
-# now only the columns correspond to chordoma samples
-smarcb1_expression <- smarcb1_expression[, which(colnames(expression_data) %in% chordoma_samples) ]
-smarcb1_expression
+smarcb1_expression <- subset_expression_data[which(rownames(subset_expression_data) == "SMARCB1"), ]
 ```
 
 ```{r}
-# remove large expression matrix that's no longer needed
-rm(expression_data)
+# now only the columns correspond to chordoma samples
+smarcb1_expression <- smarcb1_expression[, which(colnames(subset_expression_data) %in% subset_metadata$Kids_First_Biospecimen_ID) ]
```

Comment:
This line looks weird (format-wise), did you run this through a styler?

Reply:
I did but it doesn't do anything with really long lines. I'd rather use dplyr::filter here anyway but was trying to keep the original code here.